src/hssm/hssm.py

```diff
@@ -1353,9 +1353,9 @@ def plot_trace(
             Whether to call plt.tight_layout() after plotting. Defaults to True.
         """
         data = data or self.traces
-        assert isinstance(data, az.InferenceData), (
-            "data must be an InferenceData object."
-        )
+        assert isinstance(
+            data, az.InferenceData
+        ), "data must be an InferenceData object."
```

Comment:
I'd prefer throwing more informative error types like `TypeError` instead of using assertions here